plugins/inputs/internet_speed/README.md

```diff
@@ -23,9 +23,10 @@ It collects latency, download speed and upload speed
 | Download Speed | download   | float64 | Mbps |
 | Upload Speed   | upload     | float64 | Mbps |
 | Latency        | latency    | float64 | ms   |
+| Sponsor        | provider   | text    | --   |
 
 ## Example Output
 
 ```sh
-internet_speed,host=Sanyam-Ubuntu download=41.791,latency=28.518,upload=59.798 1631031183000000000
+internet_speed,host=Sanyam-Ubuntu download=41.791,latency=28.518,upload=59.798 1631031183000000000,provider=CenturyLink
```

Comment:
This cannot be an actual output as it is invalid line-protocol. ;-) The last field (separated by a space) is the timestamp so you have to move the new tag before the last space.

Reply:
Also change `sh` to `text`.